Projects/DesignSystem/Sources/Component/Extension/Date+sugar.swift

```diff
@@ -9,7 +9,14 @@
 import Foundation
 
 extension Date {
-    func convertToString() -> String {
+    func ListCardDeadlineString() -> String {
```

Comment:
```suggestion
    func hourString() -> String {
```

조금더 일반적인 네이밍이 좋을 것 같아요!

Reply:
제안해주신 방향이 좀 더 명확한거같네요! 제안 감사합니다!